packages/app/src/components/LoginForm.jsx

```diff
@@ -170,6 +171,12 @@ class LoginForm extends React.Component {
             {t('page_register.notice.restricted_defail')}
           </p>
         )}
+        { (!isMailerSetup && isEmailAuthenticationEnabled === true) && (
```

Comment:
please change to the following
```suggestion
        { (!isMailerSetup && isEmailAuthenticationEnabled) && (
```